app/Command/Publish.hs

```diff
@@ -13,6 +13,11 @@ import           Language.PureScript.Publish.ErrorsWarnings
 import           Options.Applicative (Parser)
 import qualified Options.Applicative as Opts
 
+manifestPath :: Parser FilePath
```

Comment:
Did you consider having it read from stdin instead of using a file path? I'm not actually sure which approach I prefer now.